tark/tark_web/templates/mane_collaboration_about.html

```diff
@@ -157,6 +157,13 @@ <h4>MANE Plus&nbsp;<span class="label label-info">Future project</span></h4>
                 </ul>
 
 
+                <h2>Publication</h2>
+                <p>Morales, J., Pujar, S., Loveland, J.E. et al. A joint NCBI and EMBL-EBI transcript set for clinical genomics and research.</br>
+                    Nature 604, 310–315 (2022).</br>
+                <a href="https://doi.org/10.1038/s41586-022-04558-8" target="_blank">https://doi.org/10.1038/s41586-022-04558-8</a></p>
```

Comment:
It would be good to add `rel="noopener"` for [security reasons](https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable).

Reply:
Thanks, @bilalebi. Added the requested changes. Also updated dependencies versions and using the correct version of Python for GitHub workflows